create-module.sh

```diff
@@ -263,6 +263,14 @@ cat > $readme << EOF
 # Canvas Kit CSS $upperName
 EOF
 
+# Adds module dependency and export for React and CSS
```

Comment:
This part of the conditional is the `then` if they've replied they want to add CSS module (not visible in the diff)